take-your-first-steps-with-rust/get-started-with-rust/includes/7-summary.md

```diff
@@ -1,7 +1,12 @@
 # Summary
 
-In this module, you learned...
+In this module, you learned:
```

Comment:
```suggestion
In this module, you learned what Rust is, how to install it, and how to write your first program with Rust and Cargo.
```

Reply:
I am unsure about this suggestion.

Should I remove the bulleted-list below this and adopt a prose style to list the learned topics?

Reply:
Sorry, Tiago. I had to re-read and I may have misunderstood before. Let's leave your bulleted list as is. I'll delete my comment.